app/controllers/components/course/controller_component_host.rb

```diff
@@ -122,31 +121,15 @@ def disabled_components
   def instance_enabled_components
     @instance_enabled_components ||= begin
       all_components = Course::ControllerComponentHost.components
-      all_components.select do |m|
-        enabled = @instance_settings.settings(m.key).enabled
-        enabled.nil? ? m.enabled_by_default? : enabled
-      end
-    end
-  end
-
-  # Returns the enabled components in Course, depending on the gamified flag in Course.
-  #
-  # @return [Array<Class>] array of enabled components in Course
-  def course_enabled_components
```

Comment:
@weiqingtoh I removed this because there's already an `enabled_components`, which returns the enabled course components after settings are applied.

This method is potentially confusing because the settings is not applied here.